src/ucp/wireup/wireup_cm.c

```diff
@@ -719,9 +747,7 @@ static unsigned ucp_ep_cm_disconnect_progress(void *arg)
             ucs_assert(ucp_ep->flags & UCP_EP_FLAG_CLOSED);
         }
     } else if (ucp_ep->flags & UCP_EP_FLAG_LOCAL_CONNECTED) {
-        /* if the EP is local connected, need to flush it from main thread first */
```

Comment:
what is changed here?

Reply:
we're doing error handling in `ucp_ep_cm_remote_disconnect_progress()` if no `EP_REMOTE_CONNECTED` set or `EP_CLOSED` set

Reply:
I cant find any `REMOTE_CONNECTED` handling in `ucp_ep_cm_remote_disconnect_progress`, could you point me pls?

Reply:
https://github.com/openucx/ucx/blob/8cd4d61815f5925856cca10cff6b3a5044f462b8/src/ucp/wireup/wireup_cm.c#L691